### ISSUE TITLE
Collapsed rows

### ISSUE THREAD

**Issue by suweb** [maxazan/jquery-treegrid#75]
Hi, can you advise me how to collapsed all rows by default?

**Comment by liuweiye** [comment]
> 嗨，您能告诉我默认情况下如何折叠所有行吗？
hi，as shown in the figure 
![image](https://user-images.githubusercontent.com/48922981/90145561-a3307300-ddb2-11ea-94b4-3a24ac84cf42.png)

**Comment by maxcreeger** [comment]
Yeah, from [the documentation](https://maxazan.github.io/jquery-treegrid/):
<html>
<body>


Parameter | Type | Default value | Description
-- | -- | -- | --
initialState | String | expanded | Initial state of tree'expanded' - all nodes will be expanded'collapsed' - all nodes will be collapsed

</body>
</html>


Or individually per node, apply either "treegrid-expanded" or "treegrid-collapsed" to your element' CSS class:
Ex:
```
<tr class="treegrid-110 treegrid-parent-109 treegrid-expanded">
      <td>Stuff</td>
</tr>
```
